pkg/tenant/tenant_test.go

```diff
@@ -53,6 +55,9 @@ func (suite *tenantTestSuite) SetupSuite() {
 	// Ensure Quarterdeck returns a 200 on status so Tenant knows it's ready
 	suite.quarterdeck.OnStatus(mock.UseStatus(http.StatusOK))
 
+	// Start a server to handle mock requests to Ensign
+	suite.ensign = emock.New(nil)
```

Comment:
The `New` method doesn't return an error. Benjamin confirmed that it doesn't need to since there aren't any side effects and an interface isn't required.